robots/cmd/perf-report-creator/OWNERS

```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - sig-scale-reviewers
```

Comment:
@dhiller I wonder if `- ci-maintainers` should be an explicit item here or will it be inherited from top level permissions?

Reply:
The [docs say that they are inherited](https://docs.prow.k8s.io/docs/components/plugins/approve/approvers/#overview), but looking at the top level files, I think the setup is not correct.

Reply:
Should be fixed with https://github.com/kubevirt/project-infra/pull/2779

Reply:
looks like #2779 merged, I'll rebase and push